React-Template-master/src/i18n/fr/translation.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Make user all the keys in the english and french are present in both the files

Reply:
okay